covfee/client/tasks/continuous_annotation/conflab_participant_selection.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { useEffect, useRef } from "react"
+// You can download this image and place it in the art folder from:
+// @covfee.ewi.tudelft.nl:/home/kfunesmora/conflab-media/
```

Comment:
I don't think you want to commit this path, same below

Reply:
Ok. I won't. My assumption was that access is restricted anyway and there isn't much info given there.